_posts/2022-12-6-uma_outreachy.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: Outreachy Report - 2022 Improve MDAnalysis by implementing type hinting
+---
+
```

Comment:
Add one paragraph introducing yourself (link to a personal page/GitHub/LinkedIn/...) and your Outreach project; link to Outreachy. Say when you did the project.

Readers should get an idea what an outreach project is, who you are, and what your project was about. Your post will also be the first real mentioning of Outreachy on our site so it will come up when people search for "outreachy mdanalysis". Make it look welcoming :-).